docs/rules/jsx-no-leaked-render.md

```diff
@@ -151,6 +151,10 @@ const Component = ({ elements }) => {
 
 The supported options are:
 
+### `ignoreAttributes`
+
+*TODO*
+
```

Comment:
@aleclarson hey, we need it badly, any chance reviving the draft? 🙏 Let's discuss this variant:

````suggestion
### `ignoreAttributes`
Boolean. When set to `true`, this option ignores all attributes except for `children` during validation, preventing false positives in scenarios where these attributes are used safely or validated internally. Default is `false`.

It can be set like:
```json5
{
  // ...
  "react/jsx-no-leaked-render": [<enabled>, { "ignoreAttributes": true }]
  // ...
}
```

Example of incorrect usage with default setting (`ignoreAttributes: false`) and the rule enabled (consider `value` might be undefined):

```jsx
function MyComponent({ value }) {
  return (
    <MyChildComponent nonChildrenProp={value && 'default'}>
      {value && <MyInnerChildComponent />}
   </MyChildComponent>
  );
}
```

This would trigger a warning in both `nonChildrenProp` and `children` props because `value` might be undefined.

By setting `ignoreAttributes` to `true`, the rule will not flag this scenario in `nonChildrenProp`, reducing false positives, **but will keep the warning of `children` being leaked**.
````

Reply:
@ljharb cc